lua/autorun/sh_init.lua

```diff
@@ -1,3 +1,5 @@
+AddCSLuaFile()
```

Comment:
This actually isn't necessary I think.

The renaming/moving of file was what did the houdini